diff --git a/dist/AuthenticationClient.js b/dist/AuthenticationClient.js
index a7144534a4c27b25e63804aeb4ae87a1f9c7a4e0..068250690a7b6bc4c2c5270cd603f7190a925956 100644
--- a/dist/AuthenticationClient.js
+++ b/dist/AuthenticationClient.js
@@ -314,7 +314,7 @@ class AuthenticationClient extends events_1.EventEmitter {
                         cookie: 'mobileClient=android; mobileClientVersion=777777 3.0.0'
                     },
                     deviceDetails: {
-                        device_friendly_name: 'Galaxy S22',
+                        device_friendly_name: 'Nestri Mobile',
                         platform_type: EAuthTokenPlatformType_1.default.MobileApp,
                         os_type: EOSType_1.default.AndroidUnknown,
                         gaming_device_type: 528 // dunno
diff --git a/src/AuthenticationClient.ts b/src/AuthenticationClient.ts
index 9e1c5bcf8b36d304d313c5b35795b4b543da27dd..17bce1acc689c7210060e5cea196ca63e973355b 100644
--- a/src/AuthenticationClient.ts
+++ b/src/AuthenticationClient.ts
@@ -414,7 +414,7 @@ export default class AuthenticationClient extends EventEmitter {
 						cookie: 'mobileClient=android; mobileClientVersion=777777 3.0.0'
 					},
 					deviceDetails: {
-						device_friendly_name: 'Galaxy S22',
+						device_friendly_name: 'Nestri Mobile',
 						platform_type: EAuthTokenPlatformType.MobileApp,
 						os_type: EOSType.AndroidUnknown,
 						gaming_device_type: 528 // dunno